src/baguetteBox.js

```diff
@@ -316,6 +325,8 @@
             options.animation === 'slideIn' ? '' : 'none');
         // Hide buttons if necessary
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
+            previousButtonContent.setAttribute('aria-hidden', 'true');
+            nextButtonContent.setAttribute('aria-hidden', 'true');
             options.buttons = false;
         }
         // Set buttons style to hide or display them
```

Comment:
Actually, I've overlooked the case when a user sets `options.buttons` to false himself. The _aria-hidden_ logic should better be on line 333. Could you insert there something like:
```js
if (options.buttons === false) {
    // set aria-hidden to true
    previousButton.style.display = nextButton.style.display = 'none';
} else {
   // we may also move setAttribute('aria-hidden', 'false') here
   // ... .display = '';
}
```